samples/template_ble/src/main.c

```diff
@@ -105,14 +109,47 @@ static void app_setup(void)
 	#endif
 }
 
+static enum { DFU_APPLICATION, SIDEWALK_APPLICATION } Application_to_start(){
```

Comment:
It looks strange (mixed enum definition with function implementation), could you write this in a proper way?

Reply:
This is anonymous enum, there is no need to define it separatelly if it is used only in one place

Reply:
Agree with Marcin, please consider refactoring, it's a good practice in main.c files to keep it simple. Also function names should start with low letter.
You might use sth like this:

```suggestion
typedef enum { APP_START_DFU, APP_START_SIDEWALK} app_start_t;

 app_start_t application_to_start(){
```

Reply:
fixed